levels/deepdngn/map00084.txt

```diff
@@ -154,6 +153,16 @@ CREATE_PARTY(PARTY15)
 
 REM ****************************************************************************
 
+IF(PLAYER1,WORKSHOP < 50)
+	ROOM_AVAILABLE(PLAYER1,WORKSHOP,1,0)
+ENDIF
+IF(PLAYER1,TRAINING < 50)
+	ROOM_AVAILABLE(PLAYER1,TRAINING,1,0)
+ENDIF
+IF_CONTROLS(PLAYER_GOOD,HORNY < 10)
+	ADD_CREATURE_TO_POOL(DRAGON,9)
```

Comment:
No, the reason why I limit blue dragons, is because they have large lair spaces and that might cause blue to expand early and reach the reapers. If a Reaper is down this is no longer a concern and blue should get all the dragons he wants again.